CHANGELOG.md

```diff
@@ -68,9 +70,10 @@
 - `[jest-transform]` Properly cache transformed files across tests ([#8890](https://github.com/facebook/jest/pull/8890))
 - `[jest-transform]` Don't fail the test suite when a generated source map is invalid ([#9058](https://github.com/facebook/jest/pull/9058))
 - `[jest-types]` [**BREAKING**] Use less `null | undefined` in config types ([#9200](https://github.com/facebook/jest/pull/9200))
-- `[jest-utils]` Allow querying process.domain ([#9136](https://github.com/facebook/jest/pull/9136))
+- `[jest-util]` Allow querying process.domain ([#9136](https://github.com/facebook/jest/pull/9136))
+- `[jest-util]` Copy built-in objects will keep their prototypes and value ([#9257](https://github.com/facebook/jest/pull/9257))
+- `[jest-util]` Avoid browser read Node.js `process` and `stream` property ([#9257](https://github.com/facebook/jest/pull/9257))
```

Comment:
@WeiAnAn can you remove these two lines from CHANGELOG.md for `jest-util` that reference this PR?